video-app/src/Service/HistoryService/AddToHistory.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios"; 
+
+const AddToHistory = async(video,featureDispatch) =>{
+  try {
+      const response = await axios.post("/api/user/history",{
+          video
+      },{
+          headers : { authorization: localStorage.getItem("token") ,}
+      })
+      featureDispatch({type: "HISTORY",payload: response.data.history})
```

Comment:
action type can have a better name like add to history, remove from history etc

Reply:
will change